src/chrome/content/rules/KSH.xml

```diff
@@ -6,6 +6,12 @@
 
 	<rule from="^http://(?:www\.)?ksh\.hu/"
 		to="https://www.ksh.hu/" />
+	
+	<!-- These pages return a 404 error when requested over HTTPS -->
+	<exclusion pattern="^http://statinfo\.ksh\.hu/Statinfo/" />
+		<test url="http://statinfo.ksh.hu/Statinfo/">
```

Comment:
```suggestion
		<test url="http://statinfo.ksh.hu/Statinfo/" />
```

Reply:
Totally forgot about this, but I've committed this change too now.